mne/viz/brain/view.py

```diff
@@ -0,0 +1,280 @@
+        label : str
+            Time label as string.
+        """
+        time_label = self._brain.data['time_label']
+
+        if isinstance(time_label, str):
+            label = time_label % time
+        elif callable(time_label):
+            label = time_label
+
+        return label
+
+
+class ColorBar(object):
+    u"""Helper class for visualizing a color bar.
+
+    Parameters
+    ----------
+    brain : Brain
+        Object with cortex mesh to be controlled by time
+        viewer.
+    """
+
+    def __init__(self, brain):
+        from bqplot import Axis, ColorScale, Figure, HeatMap, LinearScale
+        import ipyvolume as ipv
+        import ipywidgets as widgets
+
+        self._brain = brain
+        self._input_fmin = None
+        self._input_fmid = None
+        self._input_fmax = None
+        self._btn_upd_mesh = None
+        self._colors = None
+
+        if brain.data['center'] is None:
+            dt_min = brain.data['fmin']
+            dt_max = brain.data['fmax']
+        else:
+            dt_min = -brain.data['fmax']
+            dt_max = brain.data['fmax']
+
+        self._lut = brain.data['lut']
+        self._cbar_data = np.linspace(0, 1, self._lut.N)
+        cbar_ticks = np.linspace(dt_min, dt_max, self._lut.N)
+        color = np.array((self._cbar_data, self._cbar_data))
+        cbar_w = 500
+        cbar_fig_margin = {'top': 15, 'bottom': 15, 'left': 5, 'right': 5}
+        self._update_colors()
+
+        x_sc, col_sc = LinearScale(), ColorScale(colors=self._colors)
+        ax_x = Axis(scale=x_sc)
+        heat = HeatMap(x=cbar_ticks,
+                       color=color,
+                       scales={'x': x_sc, 'color': col_sc})
+
+        self._add_inputs()
+        fig_layout = widgets.Layout(width='%dpx' % cbar_w,
+                                    height='60px')
+        cbar_fig = Figure(axes=[ax_x],
+                          marks=[heat],
+                          fig_margin=cbar_fig_margin,
+                          layout=fig_layout)
+
+        def on_update(but_event):
+            u"""Update button click event handler."""
+            val_min = self._input_fmin.value
+            val_mid = self._input_fmid.value
+            val_max = self._input_fmax.value
+            center = brain.data['center']
+            time_idx = brain.data['time_idx']
+            time_arr = brain.data['time']
+
+            if not val_min < val_mid < val_max:
+                raise ValueError('Incorrect relationship between' +
+                                 ' fmin, fmid, fmax. Given values ' +
+                                 '{0}, {1}, {2}'
+                                 .format(val_min, val_mid, val_max))
+            if center is None:
+                # 'hot' or another linear color map
+                dt_min = val_min
+                dt_max = val_max
+            else:
+                # 'mne' or another divergent color map
+                dt_min = -val_max
+                dt_max = val_max
```

Comment:
As above, you can do this in two lines or three:

```
dt_max = val_max
dt_min = val_min if center is None else -val_max
```

Or at least get dt_max out of the loop